addons/mail/static/src/components/discuss/discuss_tests.js

```diff
@@ -421,6 +421,175 @@ QUnit.test('sidebar: add channel', async function (assert) {
     );
 });
 
+QUnit.test('sidebar: pinned channel 1: init with one pinned channel', async function (assert) {
+    assert.expect(3);
+
+    Object.assign(this.data.initMessaging, {
+        channel_slots: {
+            channel_channel: [{
+                channel_type: "channel",
+                id: 20,
+                name: "General",
+            }],
+        },
+    });
+    await this.start({});
+    assert.containsOnce(
+        document.body,
+        '.o_Discuss_thread[data-thread-model="mail.box"][data-thread-id="inbox"]',
+        "The inbox is displayed"
+    );
+    assert.containsOnce(document.body, '.o_DiscussSidebar_list .o_DiscussSidebarItem', "1 channel is present");
+    assert.strictEqual(
+        document.querySelector(`.o_DiscussSidebar_list .o_DiscussSidebarItem .o_DiscussSidebarItem_name`).textContent,
```

Comment:
keep selector simple, `. o_DiscussSidebarItem` and `. o_DiscussSidebarItem_name` are enough
(this applies to whole diff)